website/content/en/docs/kubebuilder/quickstart.md

```diff
@@ -260,6 +260,103 @@ The `ClusterRole` manifest at `config/rbac/role.yaml` is generated from the abov
 $ make manifests
 ```
 
+### Create a validating or mutating Admission Webhook (optional)
```

Comment:
I was wondering if we want to move this, and the `Update the size to an even number` sections below to a separate doc i.e `website/content/en/docs/kubebuilder/webhooks.md` and link it at the end of this doc?

That would help keep the quickstart brief for folks looking at it for the first time, and keeps the webhooks doc separate as an advanced follow up.

Reply:
@hasbro17 done